packages/synapse-react-client/src/components/ReleaseCard/ReleaseCardTypes.ts

```diff
@@ -0,0 +1,50 @@
+export type ReleaseCardStat = {
+  label: string
+  value: string
+}
+
+export type StatConfig = {
+  // specifies which column should be used to populate the statistic value
+  columnName: string
+  // the label that will be displayed with the statistic in the card
+  label: string
+}
+
+export type ButtonToExplorePageConfig = {
+  // button's label
+  label: string
+  // column name in the data (table, view...) driving the page where the button is located
+  // which is used to populate the button's path
+  sourcePathColumnName: string
+  // defining SQL for the data (table, view...) driving the Explore page
+  // which is used in generating the button's query string
+  exploreDataSql?: string
+  // column name of the selected facet in the data (table, view...) driving the Explore page
+  // which is used in generating the button's query string
+  exploreDataFacetColumnName?: string
+  // column name in the data (table, view...) driving the page where the button is located
+  // which is used to to populate the value for the selected facet in the button's query string
+  sourceDataFacetValueColumnName?: string
+}
```

Comment:
In the GENIE portal, ["current release" cards](https://www.figma.com/file/BI4y33EHA95onN8DourTNZ/Two-Projects?type=design&node-id=275-22889&mode=design&t=76oHvfvp9FWFtDSR-4) link to different pages within the Portal depending on release type. For GENIE releases, the explore button redirects to `/explore/files/genie`. For other types of releases, the explore button redirects to `/explore/files/genie-bpc` with the current release type facet selected (e.g.'crc'). 

I used this config to specify how the button's link should be built using the table that drives the Release Cards, e.g. https://www.synapse.org/#!Synapse:syn53701326/tables/, but happy to adjust if there's a better/established way to dynamically construct these types of links! 

*The Figma only shows one current release, but there may be multiple current releases shown on the same page, as shown in [the ticket](https://sagebionetworks.jira.com/browse/PORTALS-2961).